kubelogin.rb

```diff
@@ -4,8 +4,8 @@ class Kubelogin < Formula
   version "v1.23.0"
```

Comment:
This version needs to also been bumped. Currently it already diverged, meaning that `brew` says it install `1.23.0`. but it is actually `1.23.1`

Reply:
@int128 Could you have a look?